versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md

```diff
@@ -153,27 +153,34 @@ next to the status of a **Finished** job to visualize the results.
 
   ![image](../../assets/monitoring/discovery/host-discovery-hosts-listing.png)
 
-2. If you want, you can edit the mappers linked to this job by clicking the **Edit** icon: ![image](../../assets/monitoring/discovery/host-discovery-edit.png#thumbnail1)
+  The results show the discovered hosts. Icons allow you to identify the action that will be performed on the host:
+   
+   - The **plus** icon means the host does not exist yet and will be created.
+   - The **arrow** icon means the host already exists and its configuration will be updated.
 
-    The results will be updated automatically in the preview once you click on the **Save** icon at the top right of the panel.
+2. Select the hosts you want to add to the list of monitored hosts and click the **Save** button. Then click the **Back** button to return to the **Discovery** page.
 
-    - If the job was set to **Manual analysis** at step 5 of the wizard, bear in mind that editing mappers after a discovery job has been executed only makes sense if the hosts have not yet been added to your monitoring configuration.
+3. If you want, you can edit the mappers linked to this job. Click on the job to display its properties.  In the **Mappers** section, edit the mapper with its **Edit** icon: ![image](../../assets/monitoring/discovery/host-discovery-edit.png#thumbnail1)
 
-    - If the job was set to **Automatic analysis** at step 5 of the wizard and **Update existing hosts** was selected, the hosts will be updated when you re-run the job (see [Edit a discovery job](#edit-a-discovery-job)).
+  The results will be updated automatically in the preview once you click on the **Save** icon at the top right of the panel.
+  
+   - If the job was set to **Manual analysis** at step 5 of the wizard, bear in mind that editing mappers after a discovery job has been executed only makes sense if the hosts have not yet been added to your monitoring configuration.
```

Comment:
This warning makes no sense now that the hosts can be updated in manual mode ;-)
I suggest removing it